DigitalLearningSolutions.Data/DataServices/UserDataService/UserCentreDetailsDataService.cs

```diff
@@ -71,8 +72,8 @@ public bool EmailIsInUseByOtherUser(int userId, string email)
         {
             return connection.QuerySingle<bool>(
                 @"SELECT CASE
-                        WHEN EXISTS (SELECT ID FROM USERS WITH (UPDLOCK, TABLOCKX, HOLDLOCK) WHERE PrimaryEmail = @email AND ID <> @userId)
-                        OR EXISTS (SELECT ID FROM UserCentreDetails d WITH (UPDLOCK, TABLOCKX, HOLDLOCK)
+                        WHEN EXISTS (SELECT ID FROM USERS WHERE PrimaryEmail = @email AND ID <> @userId)
+                        OR EXISTS (SELECT ID FROM UserCentreDetails d
                             WHERE d.Email = @email AND d.Email IS NOT NULL AND d.UserID <> @userId)

```

Comment:
Just noticed we've got `AND d.Email IS NOT NULL` -  I don't think we want that. If `email` is not null then `d.Email = email` implies `d.Email` is not null. If we do pass `email = null` into this method then the method should return true if there are other users with null emails (or that's what its name claims). Same in method below.